ursula_cli/shell.py

```diff
@@ -427,6 +430,9 @@ def main():
     parser.add_argument('--provisioner',
                         help='The external provisioner to use',
                         default=None, choices=["vagrant", "heat"])
+    parser.add_argument('--heat-stack-name',
+                        help='Name of the heat stack when heat provisioner is used',
+                        default=None)
```

Comment:
So I think I'm :smoking:  crack because args.environment won't be available yet since the args have not be processed yet.